scss/tech.scss

```diff
@@ -1,3 +1,53 @@
+@import 'colors';
+@import 'fonts';
+
+.tech-container {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: auto;
+  margin: 20px;
+  padding: 10px 20px 20px 10px;
+  position: relative;
+  text-align: center;
+  width: 220px;
+
+  &::before {
+    bottom: 10px;
+    border: 1px solid;
+    border-color: $earls-green;
+    border-radius: 5px;
+    content: '';
+    height: 100%;
+    position: absolute;
+    right: 10px;
+		width: 100%;
```

Comment:
Almost there with indentation >.<

Reply:
I dont know whats going on with this, in my file is ok. Will try something, hope it works.